runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -147,13 +150,21 @@ public int getSizeEstimate() {
         return 1;
     }
 
+    private int calculateMetaDataMapSize() {
+        int size = 0;
+        for (Map.Entry<LogUnitMetadataType, Object> entry : metadataMap.entrySet()) {
+            entry.
+        }
```

Comment:
Is something missing here?

Reply:
yes, it was to be deleted.  Removed now.  Thanks.